xml/deploy_intro.xml

```diff
@@ -14,22 +14,11 @@
   </dm:docmanager>
  </info>
  <para>
-  Since &productname; 7, &ceph; services are deployed as containers instead of
-  packages. There are two ways to deploy the cluster:
- </para>
- <itemizedlist>
-  <listitem>
-   <para>
-    By using &cephadm;. Find more details in <xref linkend="deploy-cephadm"/>.
-   </para>
-  </listitem>
-  <listitem>
-   <para>
-    On top of a &caasp; cluster. Find more details in
-    <xref linkend="deploy-rook"/>.
-   </para>
-  </listitem>
- </itemizedlist>
+  Since &productname; &productnumber;, &ceph; services are deployed as containers
```

Comment:
I think we can use a specific hard-wired number here, since it won't change over time.